SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -277,6 +277,11 @@ class CFVMFlowSolverBase : public CSolver {
    */
   void SumEdgeFluxes(const CGeometry* geometry);
 
+  /*!
+   * \brief Computes and sets the required auxilliary vars (and gradients) for axisymmetric flow.
+   */
+  void PrepareAndConputeAuxVarGrad(CGeometry *geometry, const CConfig* config);
```

Comment:
```suggestion
  void ComputeAxisymmetricAuxGradients(CGeometry *geometry, const CConfig* config);
```
I'm not aware of the proper naming but let's go for something more descriptive.